src/webots/scene_tree/WbSceneTree.cpp

```diff
@@ -754,19 +754,6 @@ void WbSceneTree::convertProtoToBaseNode(bool rootOnly) {
     if (skipTemplateRegeneration)
       parentField->blockSignals(false);
 
-    // copy resources (textures and meshes)
-    QHashIterator<QString, QString> it(writer.resourcesList());
-    while (it.hasNext()) {
-      it.next();
-      const QString destination(WbProject::current()->worldsPath() + it.key());
-      if (!(WbUrl::isLocalUrl(it.key()) || WbUrl::isWeb(it.key()))) {
-        const QFileInfo fileInfo(destination);
-        if (!QDir(fileInfo.absolutePath()).exists())
-          QDir().mkpath(fileInfo.absolutePath());
-        QFile::copy(it.value(), destination);
-      }
-    }
-
```

Comment:
Why is it not needed anymore? (if you want to convert a local proto)

Reply:
Because now the isProto writer leverages the same logic of the x3d writer and expresses the URL relative to the world so, when converting to basenode, locally defined assets are re-written relative to the world therefore there isn't (shouldn't) be a need to create copies of the textures/meshes